src/components/pages/search-page/SearchPage.jsx

```diff
@@ -17,37 +18,69 @@ const rows = [
   },
 ]
 
+const formikState = {
+  id: '',
+  phone: '',
+  city: '',
+  streetName: '',
+  house: '',
+}
+
 const SearchPage = () => {
   return (
     <>
-      <form action="" className="search-page-form">
-        <LabelledInput
-          label="ID заказа"
-          labelClassName="search-page-form-field"
-          centered
-        />
-        <LabelledInput
-          label="Телефон"
-          labelClassName="search-page-form-field"
-          centered
-        />
-        <LabelledInput
-          label="Город"
-          labelClassName="search-page-form-field"
-          centered
-        />
-        <LabelledInput
-          label="Улица"
-          labelClassName="search-page-form-field"
-          centered
-        />
-        <LabelledInput
-          label="Дом"
-          labelClassName="search-page-form-field"
-          centered
-        />
-        <Button className="search-page-form-submit-button">Найти</Button>
-      </form>
+      <Formik
+        initialValues={formikState}
+        onSubmit={(values) => {
+          // eslint-disable-next-line no-console
+          console.log(JSON.stringify(values, null, 2))
+        }}
+      >
+        {({ handleChange }) => (
+          <>
+            <Form action="" className="search-page-form">
+              <LabelledInput
+                label="ID заказа"
+                labelClassName="search-page-form-field"
+                centered
+                id="id"
```

Comment:
для связи инпута с formik state можно использовать и id, и name, но `name` обычно служит именно для этой цели, поэтому давай здесь и в других местах будем использовать `name`, а не `id`
```suggestion
                name="id"
```